bake/compose.go

```diff
@@ -111,3 +119,74 @@ func flatten(in compose.MappingWithEquals) compose.Mapping {
 	}
 	return out
 }
+
+// extToTarget converts Compose build extension x-bake to bake Target
+// https://github.com/compose-spec/compose-spec/blob/master/spec.md#extension
+func extToTarget(exts map[string]interface{}) (*Target, error) {
+	t := new(Target)
+	if ext, ok := exts["x-bake"]; ok {
+		for key, val := range ext.(map[string]interface{}) {
+			switch key {
+			case "tags":
+				if res, k := val.(string); k {
+					t.Tags = append(t.Tags, res)
+				} else {
+					for _, res := range val.([]interface{}) {
+						t.Tags = append(t.Tags, res.(string))
+					}
+				}
+			case "cache-to":
```

Comment:
you can restore `cache-from` but make sure it overrides the general field if set.